scipy-1.0/core-dev.tex

```diff
@@ -0,0 +1,66 @@
+\author[1]{Pauli Virtanen}
+\author[2,*]{Ralf Gommers}
+\author[3,4,5,6,2]{Travis E. Oliphant}
+\author[7,8,*]{Matt Haberland}
+\author[9,*]{Tyler Reddy}
+\author[10]{David Cournapeau}
+\author[11]{Evgeni Burovski}
+\author[12,13]{Pearu Peterson}
+\author[10]{Warren Weckesser}
+\author[14]{Jonathan Bright}
+\author[15]{St\'efan J. van der Walt}
+\author[16]{Matthew Brett}
+\author[10]{Joshua Wilson}
+\author[17,18]{K. Jarrod Millman}
+\author[19]{Nikolay Mayorov}
+\author[20]{Andrew R. J. Nelson}
+\author[5]{Eric Jones}
+\author[5]{Robert Kern}
+\author[21]{Eric Larson}
+\author[22]{CJ Carey}
+\author[10]{\.{I}lhan Polat}
+\author[23]{Yu Feng}
+\author[24]{Eric W. Moore}
+\author[25]{Jake VanderPlas}
+\author[10]{Denis Laxalde}
+\author[10]{Josef Perktold}
+\author[26]{Robert Cimrman}
+\author[27]{Ian Henriksen}
+\author[10]{E. A. Quintero}
+\author[10]{Charles R Harris}
+\author[28]{Anne M. Archibald}
+\author[29]{Ant\^{o}nio H. Ribeiro}
+\author[30]{Fabian Pedregosa}
+\author[30]{Paul van Mulbregt}
+\author[ ]{SciPy 1.0 Contributors}
+\affil[1]{University of Jyv\"askyl\"a, FI-40014 University of Jyv\"askyl\"a, Finland}
+\affil[2]{Quansight LLC, Austin, TX, USA}
+\affil[3]{Ultrasound Imaging, Mayo Clinic, Rochester, MN 55902, USA}
+\affil[4]{Electrical Engineering, Brigham Young University, Provo, UT 84604, USA}
+\affil[5]{Enthought, Inc., Austin, TX 78701, USA}
+\affil[6]{Anaconda Inc, Austin, TX 78701, USA}
+\affil[7]{BioResource and Agricultural Engineering Department, California Polytechnic State University, San Luis Obispo, CA 93407, USA}
+\affil[8]{Department of Mathematics, University of California, Los Angeles, CA 90025, USA}
+\affil[9]{Los Alamos National Laboratory, Los Alamos, NM 87544}
+\affil[10]{Independent Researcher}
+\affil[11]{Higher School of Economics, Myasnitskaya 20, Moscow 101000 Russia}
+\affil[12]{Independent Researcher, Saue, Estonia}
+\affil[13]{Department of Mechanics and Applied Mathematics, Institute of Cybernetics at Tallinn Technical University, Akadeemia Rd 21 Tallinn, Estonia}
+\affil[14]{Independent Researcher, NY, USA}
+\affil[15]{Berkeley Institute for Data Science, University of California Berkeley, Berkeley, CA 94720 USA}
+\affil[16]{School of Psychology, University of Birmingham, Edgbaston, Birmigham B15 2TT, UK}
+\affil[17]{Berkeley Institute for Data Science, University of California, Berkeley, CA 94720 USA}
```

Comment:
looks like you got a duplicate between 15 & 17 here, didn't check the rest...

Reply:
Perhaps a little more painful to fix this one--the cascading effect of removing one of the affiliation numbers may be ugly.

Reply:
Note most entries seem to have a comma before country;  the BIDS entries don't.